<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewIncome</class>
 <widget class="QDialog" name="AddNewIncome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_propertNickname_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Add Income</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formAddExpense_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboProperty">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>Select Property</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelIncomeDate">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Income Date:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dateIncome">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelIncomeDescription_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Income Description:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboDescription">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>Select Description</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelIncomeAmount">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Income Amount:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinIncome">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelIncomeReceived">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Received By:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboReceived">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>Select Payer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Property:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushAddIncome_2">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
